base/dates/io.jl

```diff
@@ -429,7 +434,25 @@ end
 
 Parse a date from a date string `dt` using a `DateFormat` object `df`.
 """
-Date(dt::AbstractString,df::DateFormat=ISODateFormat) = parse(Date, dt, df)
+Date(dt::AbstractString, df::DateFormat=ISODateFormat) = parse(Date, dt, df)
+
+"""
+    Time(dt::AbstractString, format::AbstractString; locale="english") -> Time
+
+Construct a `Time` object by parsing a `dt` time string following the pattern given in the
+`format` string. Follows the same conventions as
+`DateTime(::AbstractString, ::AbstractString)`.
+"""
+function Time(dt::AbstractString, format::AbstractString; locale::Locale=ENGLISH)
```

Comment:
Probably should not use `dt` (stands for Date Time) for the variable name. Maybe `t` is better